labgrid/driver/qemudriver.py

```diff
@@ -141,6 +141,9 @@ def on_activate(self):
                 "if=pflash,format=raw,file={},id=nor0".format(
                     self.target.env.config.get_image_path(self.flash)))
 
+        if "-append" in self.extra_args:
```

Comment:
Maybe use in self.extra_args.split() here to avoid detecting an "-append" in other arguments?

Reply:
You're right. I think `shlex.split()` is even better. It allows theoretical constructs like `extra_args: '-object "foo -append bar"'` in the yaml.

Reply:
Force-pushed `shlex.split()` variant.